loky/backend/_win_wait.py

```diff
@@ -38,7 +40,13 @@ def wait(connections, timeout=None):
 
     while True:
         # We cannot use select as in windows it only support sockets
-        ready = [c for c in connections if c.poll(0)]
+        ready = []
+        for h in handles:
+            if type(h) == int:
+                if WaitForSingleObject(h, 0) == WAIT_OBJECT_0:
+                    ready += [h]
+            elif h.poll(0):
+                ready.append(h)
```

Comment:
It is a degraded backport from `wait` in 3.3 with a bit of exploration of the window API.